website/content/docs/security/acl/acl-endpoints-by-resource.mdx

```diff
@@ -0,0 +1,222 @@
+---
+layout: docs
+page_title: Consul API Endpoint Access List
+description: |-
+  Consul API Access list based on the action and access level
+---
+
+# API Endpoint Access List
+
+Below are lists of API endpoints organized by the resource and access level that controls. Write access also provides read access.
+
+## ACL
+Write includes Read permissions
+
+<Tabs>
+
+<Tab heading="Read">
+
+@include 'endpoints_acl_read.mdx'
+
+</Tab>
+
+<Tab heading="Write">
+
+@include 'endpoints_acl_write.mdx'
+
+</Tab>
+
+</Tabs>
+
+## Agent
+Write includes Read permissions
+
+<Tabs>
+
+<Tab heading="Read">
+
+@include 'endpoints_agent_read.mdx'
+
+</Tab>
+
+<Tab heading="Write">
+
+@include 'endpoints_agent_write.mdx'
+
+</Tab>
+
+</Tabs>
+
+## Event
+Write includes Read permissions
+
+<Tabs>
+
+<Tab heading="Read">
+
+@include 'endpoints_event_read.mdx'
+
+</Tab>
+
+<Tab heading="Write">
+
+@include 'endpoints_event_write.mdx'
+
+</Tab>
+
+</Tabs>
+
+## Intentions
+Write includes Read permissions
+
+<Tabs>
+
+<Tab heading="Read">
+
+@include 'endpoints_intentions_read.mdx'
+
+</Tab>
+
+<Tab heading="Write">
+
+@include 'endpoints_intentions_write.mdx'
+
+</Tab>
+
+</Tabs>
+
+## Key
+Write includes Read permissions
+
+<Tabs>
+
+<Tab heading="Read">
```

Comment:
It looks like the special `list` policy disposition only exists for the `key` resource (see https://www.consul.io/docs/security/acl/acl-rules#policy-dispositions).

If I understand correctly, `list` is just a recursive version of `read`, so it would have the same privileges as `read. Maybe the table heading can be "Read or List"?